requirements.txt

```diff
@@ -33,13 +33,13 @@ pypssl==2.2
 pysafebrowsing==0.1.1
 PySocks==1.7.1
 python-magic==0.4.22
-quark-engine==21.6.3
+git+https://github.com/quark-engine/quark-engine.git@v21.7.1 #pip version has conflict with celery
```

Comment:
https://github.com/intelowlproject/IntelOwl/pull/549

Reply:
@0ssigeno

Reply:
Issue should be still present in that version

Reply:
We can wait for a version of Quark that solves this issue. I don't think we are in a hurry to merge this pr